app/controllers/comments_controller.rb

```diff
@@ -13,6 +13,20 @@ def create
     end
   end
 
+  def edit
+    @comment = Comment.find(params[:id])
```

Comment:
Can anyone edit a comment? You probably want to scope this to the user who created the comment:

```ruby
@comment = Comment.where(user: current_user, id: params[:id]).first

# or the simplified, preferred version:

@comment = current_user.comments.find(params[:id])
```